conf/carbon.properties.template

```diff
@@ -110,7 +100,7 @@ carbon.enable.quick.filter=false
 ##Percentage to identify whether column cardinality is more than configured percent of total row count
 #high.cardinality.row.count.percentage=80
```

Comment:
This is also not used I guess, please check and remove

Reply:
not used  removed